addons/arsenal/functions/fnc_compileSorts.sqf

```diff
@@ -0,0 +1,82 @@
+#include "script_component.hpp"
+/*
+ * Author: SynixeBrett
+ * Create the internal stats arrays when needed for the first time
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+*/
+
+if (!isNil QGVAR(sortListLeftPanel)) exitWith {};
+
+private _fnc_addToTabs = {
+    params ["_tabsList", "_tabsToAddTo", "_sideString"];
+    {
+        private _arrayToSave = +_finalArray;
+        _arrayToSave set [0, ([_class, _sideString, [str _x, format ["0%1", _x]] select (_x < 10)] joinString "")];
+        (_tabsList select _x) pushBack _arrayToSave;
+    } foreach _tabsToAddTo;
+};
+
+private _sortListLeftPanel = [
+    [], // Primary 0
+    [], // Handgun 1
+    [], // Launcher 2
+    [], // Uniform 3
+    [], // Vests 4
+    [], // Backpacks 5
+    [], // Headgear 6
+    [], // Goggles 7
+    [], // NVGs 8
+    [], // Binoculars 9
+    [], // Map 10
+    [], // GPS 11
+    [], // Radio 12
+    [], // Compass 13
+    [] // Watch 14
+];
+
+private _sortListRightPanel = [
+    [], // Optics 0
+    [], // Side accs 1
+    [], // Muzzle 2
+    [], // Bipod 3
+    [], // Mag 4
+    [], // Throw 5
+    [], // Put 6
+    []  // Misc 7
+];
+
+//------------------------- Config handling
+private _configEntries = "(getNumber (_x >> 'scope')) == 2" configClasses (configFile >> QGVAR(sorts));
+
+{
+    private _finalArray = [];
```

Comment:
Unnecessary, can just add the `private` to L70?

Reply:
This is how stats did it, if we change it here it should be changed in stats too